front-end/src/app/login/login.component.ts

```diff
@@ -10,7 +10,9 @@ import {ActivatedRoute, Router} from '@angular/router';
 
 export class LoginComponent implements OnInit {
 
-  model: any = {};
+  email : string;
```

Comment:
These should have @Input() tag, but I think Tycho didn't commit yet so it will be updated soon.